src/main/java/myApp/controller/OrderController.java

```diff
@@ -0,0 +1,29 @@
+package myApp.controller;
+
+import myApp.entity.Order;
+import myApp.repository.OrderRepository;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
better to make it "/api/orders" cause orders is a area of ​​responsibility of this controller